File: GTTS/INL.MessagingService.Client/Models/INotificationRosterUploaded_Item.cs
using System.Collections.Generic;

namespace INL.MessagingService.Client.Models
{
    interface INotificationRosterUploaded_Item
    {
        long TrainingEventID { get; set; }
        string Name { get; set; }
        int OrganizerAppUserID { get; set; }
        string UploadedBy { get; set; }
        string EventOverviewURL { get; set; }
        List<NotificationRecipientNamesObj> KeyParticipants { get; set; }
        List<NotificationRecipientNamesObj> UnsatisfactoryParticipants { get; set; }
        List<AppUserIDObj> Stakeholders { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.MessagingService.Models/inotification-roster-uploaded_item.ts



import { NotificationRecipientNamesObj } from './notification-recipient-names-obj';
import { AppUserIDObj } from './app-user-idobj';

export interface INotificationRosterUploaded_Item {
  
	TrainingEventID: number;
	Name: string;
	UploadedBy: string;
	KeyParticipants?: NotificationRecipientNamesObj[];
	UnsatisfactoryParticipants?: NotificationRecipientNamesObj[];
	Stakeholders?: AppUserIDObj[];

}

